.go-version

```diff
@@ -1 +1 @@
-1.13.8
+1.19.1
```

Comment:
If we're going to bump this, can we move to the latest patch version? I don't know if anybody is using this on Windows, but 1.19.3 contains a security fix for environment variable handling in spawned processes on Windows.